### PR TITLE
speed up saving of homefile

### PR DESCRIPTION
**Pull request by pinkysnowman**
switched to table.concat() and only use output:write() one time for speed

### CODE REVIEW

**Comment by kaeza** [comment]
Conflicts with #736, and is not needed for an operation that is not executed very often.

File: mods/sethome/init.lua [hunk 1]
```diff
@@ -54,11 +54,13 @@ minetest.register_chatcommand("sethome", {
         minetest.chat_send_player(name, "Home set!")
         changed = true
         if changed then
-        	local output = io.open(homes_file, "w")
-            for i, v in pairs(homepos) do
-                output:write(v.x.." "..v.y.." "..v.z.." "..i.."\n")
-            end
-            io.close(output)
+        	local output = io.open(home_file, "w")
+			local data = {}
```
Comment (asl97) [inline mods/sethome/init.lua]: why the indent?

File: mods/sethome/init.lua [hunk 2]
```diff
@@ -54,11 +54,13 @@ minetest.register_chatcommand("sethome", {
...
+			for n, p in pairs(homepos) do
+				table.insert(data,(p.x.." "..p.y.." "..p.z.." "..n.."\n")
```
Comment (asl97) [inline mods/sethome/init.lua]: missing closing bracket?

File: mods/sethome/init.lua (hunk 1 above)
Comment (pinkysnowman) [inline mods/sethome/init.lua]: I think my editor did that opps lol

**Comment by asl97** [comment]
@kaeza it does?  i don't see how this would conflict with #736.

**Comment by kaeza** [comment]
@asl97 see edit. I blame OctoDroid for bringing the wrong pull to top :P

File: mods/sethome/init.lua (hunk 1 above)
Comment (pinkysnowman) [inline mods/sethome/init.lua]: @kaeza just because the code is not ran alot doesn't mean that it can't be
improved upon ;)

**Comment by asl97** [comment]
@kaeza you can delete your post yourself by clicking the X button on the top right corner.

File: mods/sethome/init.lua (hunk 1 above)
Comment (asl97) [inline mods/sethome/init.lua]: @pinkysnowman imo, it would be better to leave a reply for the pull request on the pull request rather than in the note

Comment (pinkysnowman) [inline mods/sethome/init.lua]: Sorry on my mobile if i reply it does that

**Comment by kaeza** [comment]
#747

File: mods/sethome/init.lua (hunk 1 above)
Comment (ShadowNinja) [inline mods/sethome/init.lua]: This is indented with a mix of spaces and tabs.

**Pull request closed without merging**